internal/mill/exec_test.go

```diff
@@ -384,14 +402,37 @@ func newPetFixture(t *testing.T) *petFixture {
 	dir, _ := ioutil.TempDir("", t.Name())
 	wmDir := dirs.NewWindmillDirAt(dir)
 	procfs, err := proc.NewProcFSWithDir(wmDir)
+	drymode := false
+	if err != nil {
+		t.Fatal(err)
+	}
+	runner := proc.NewRunner(procfs)
+	school := school.NewPetSchool(procfs)
+	return &petFixture{
+		t:         t,
+		petsitter: NewPetsitter(stdout, stderr, runner, procfs, school, drymode),
+		stdout:    stdout,
+		stderr:    stderr,
+		dir:       dir,
+		procfs:    procfs,
+	}
+}
+
+func newPetFixtureDryRun(t *testing.T) *petFixture {
```

Comment:
instead of copying the init code, it would be easier to set fixture.petsitter.DryMode = true?